src/Resources/views/standard_layout.html.twig

```diff
@@ -9,7 +9,7 @@ file that was distributed with this source code.
 
 #}
 
-{% set _preview = block('preview') is defined ? block('preview')|trim : null %}
+{%- set _preview = block('preview') is defined ? block('preview')|trim : null %}
```

Comment:
What about the other blocks?

Reply:
They don't require the whitespace control in order to fix the issue described here, since the line breaks are printed before `{% set _preview...` and after `...block('list_filters_actions')|trim : null %}`. See [Whitespace Control](https://twig.symfony.com/doc/2.x/templates.html#whitespace-control).

Reply:
Missed the `-%}` at the end of the whole block